Python/Tests/Analysis/LanguageServerTests.cs

```diff
@@ -629,6 +629,28 @@ class C:
             await AssertHover(s, mod, new SourceLocation(16, 1), "x: int, float", new[] { "int", "float" }, new SourceSpan(16, 1, 16, 2));
         }
 
+        [TestMethod, Priority(0)]
+        public async Task HoverSpanCheck() {
+            var s = await CreateServer();
+                var mod = await AddModule(s, @"import datetime
+datetime.datetime.now().day
+");
+
+            await AssertHover(s, mod, new SourceLocation(2, 1), "built-in module datetime*", new[] { "datetime" }, new SourceSpan(2, 1, 2, 9));
+            if (Default.Version < Microsoft.PythonTools.Parsing.PythonLanguageVersion.V30) {
+                await AssertHover(s, mod, new SourceLocation(2, 11), "class datetime.datetime*", new[] { "datetime.datetime" }, new SourceSpan(2, 1, 2, 18));
+            } else {
+                await AssertHover(s, mod, new SourceLocation(2, 11), "datetime.datetime:*", new[] { "datetime", "datetime.datetime" }, new SourceSpan(2, 1, 2, 18));
+            }
+            await AssertHover(s, mod, new SourceLocation(2, 20), "datetime.datetime.now: bound built-in method now*", null, new SourceSpan(2, 1, 2, 22));
+
+            var text = Default.Version < Microsoft.PythonTools.Parsing.PythonLanguageVersion.V30
```

Comment:
We have `.Is3x()` and `.Is2x()` extension methods for these (should be in the Parsing namespace), but typically in tests we use different subclasses (see `CompletionInClassDefinition` in this class, for example).

Reply:
OK.